### DISCUSSION TITLE
Recommendations for SOAP and Quarkus 3

### DISCUSSION THREAD

**Discussion by lukaslengyelpf** [quarkusio/quarkus#37281] [Q&A]
Hello, I am trying to find out the correct way how to develop SOAP Client with Quarkus 3. I checked quarkus-quickstarts projects and guides, but no luck. I did some projects with Quarkus 2 using quarkus-jaxb and quarkus-cxf extension, but I am not sure if this is the recommended way. Thanks for any advice.

**Comment by maxandersen** [comment]
Quarkus-cxf provides a way.
